changelog/10473.feature.md

```diff
@@ -0,0 +1 @@
+Support graph recipes.
```

Comment:
Might be worth having a slightly more expansive description here. Also we can link to the docs?